tests/png-merkle-distributor.ts

```diff
@@ -1,277 +0,0 @@
-import type { Program } from "@project-serum/anchor";
```

Comment:
These tests were absolutely syntactically scuffed - I added some extra tests to the end of `merkle-distributor.spec.ts` for updating the tree